src/utils/index.js

```diff
@@ -0,0 +1,53 @@
+
+export const mapComment = commentLen => {
+  if (commentLen > 2) {
+    return `${commentLen} comments`;
+  } else if (commentLen === 1) {
+    return `${commentLen} comment`;
+  } else {
+    return `discuss`;
+  }
+};
+
+export function getSourceUrl(url) {
+  const regex = /^(?:https?:\/\/)?(?:[^@\n]+@)?(?:www\.)?([^:/\n]+)/;
```

Comment:
This looks odd, but maybe I'm just reading it wrong. I don't think the whole part with the newlines is needed, that wouldn't happen in a URL.

Also, a comment describing longer regular expressions would be appreciated.

Reply:
added comments